spa/src/api/rest-api.ts

```diff
@@ -0,0 +1,75 @@
+export function createRestApi() {
+
+	let gitHubAccessToken: string | undefined;
```

Comment:
My idea is that, assume there's some malicious code running in user browser, those malicious scripts cannot read the value of those token, like `window.gitHubToken` or `window.react.__context.PageConnect.__imortedAuthManager.gitHubAccessToken`

Reply:
Above `gitHubAccessToken` is in the "closure" of this function, so it wont' be disclosed.